src/components/CIPipelineN/TaskTypeDetailComponent.tsx

```diff
@@ -0,0 +1,366 @@
+                        />
+                        <RadioGroup
+                            className="no-border"
+                            value={
+                                formData[activeStageName].steps[selectedTaskIndex].inlineStepDetail
+                                    .mountDirectoryFromHost
+                                    ? MountPath.TRUE
+                                    : MountPath.FALSE
+                            }
+                            disabled={false}
+                            name="mountDirectoryFromHost"
+                            onChange={(event) => {
+                                handleMountChange(event, 'mountDirectoryFromHost')
+                            }}
+                        >
+                            <RadioGroupItem value={MountPath.FALSE}> {MountPath.FALSE} </RadioGroupItem>
+                            <RadioGroupItem value={MountPath.TRUE}> {MountPath.TRUE} </RadioGroupItem>
+                        </RadioGroup>
+                    </div>
+                    {formData[activeStageName].steps[selectedTaskIndex].inlineStepDetail.mountDirectoryFromHost && (
+                        <MountFromHost />
+                    )}
+                    <OutputDirectoryPath />
+                </>
+            )
+        }
+    }
+
+    const renderDockerScript = () => {
+        if (formData[activeStageName].steps[selectedTaskIndex].inlineStepDetail.scriptType === ScriptType.DOCKERFILE) {
+            return (
+                <>
+                    <div className="row-container mb-12">
+                        <div className="fw-6 fs-13 lh-32 cn-7 ">Docker path *</div>{' '}
+                        <input
+                            style={{ width: '80% !important' }}
+                            className="form__input bcn-1 w-80"
+                            autoComplete="off"
+                            placeholder="Enter Mount script path"
+                            type="text"
+                        />
+                    </div>
+                    <div className="row-container mb-12">
+                        <Tippy className="default-tt" arrow={false} content="Path where script should be mounted">
+                            <div className="fw-6 fs-13 lh-32 cn-7 ">Mount script at *</div>
+                        </Tippy>
+                        <input
+                            style={{ width: '80% !important' }}
+                            className="form__input bcn-1 w-80"
+                            autoComplete="off"
+                            placeholder="Enter Mount script path"
+                            type="text"
+                        />
+                    </div>
+                </>
+            )
+        }
+    }
+
+    return (
+        <>
+            {renderShellScript()}
+            {renderContainerScript()}
+            {/* {renderDockerScript()} */}
```

Comment:
If it's not going to be used/shown `renderDockerScript` then remove this commented code and `renderDockerScript` function.